LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+[![Hyperledger Indy (Nathan George)](../images/video-image.png)](https://youtu.be/nNo5MzgkWxE)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger Indy
+[Hyperledger Indy](https://www.hyperledger.org/projects) is a distributed ledger purpose-built for decentralized identity. Hyperledger Indy's goal is to achieve this by developing a set of
+
+>"(...) decentralized identity specs and artifacts that are independent of any particular ledger and will enable **interoperability** across any DLT that supports them."
+
+Contributed by the Sovrin Foundation, Hyperledger Indy allows individuals to manage and control their digital identities. Rather than having businesses store huge amounts of personal data of individuals, Hyperledger Indy allows businesses to store pointers to identity. Once the company verifies the other party's identity, it throws it away.
+
+According to Brian Behlendorf,
+
+>"(...) identity is a toxic asset that could present a liability to organizations."
+
+Indeed, since 2013, over 9 billion data records were lost or stolen. What is striking is that, out of these, only 4% were encrypted, and hence, rendered useless after being stolen (also called 'secure breaches'). You can find detailed statistics at http://breachlevelindex.com/.
+
+One of the key principles of Hyperledger Indy is its '[Privacy by Design](https://en.wikipedia.org/wiki/Privacy_by_design)' approach. Given the immutable nature of the DLT, it is all the more important that digital identities be handled with the utmost care, keeping human values front and center.
+
+>"Hyperledger Indy lets users authenticate identity based on the attributes they are willing to store and share themselves. This can reduce the amount of liability contained within a business because the data can be kept with the user and presented to you again in a way that you can trust and validate that what has been said really was said and is trusted by the other parties you do business with."
+
+\- Nathan George, Maintainer, Hyperledger Indy
+
+Further information about the history of the project can be found at https://sovrin.org/.
+
+## Hyperledger Burrow v0.16.1
+Formally known as eris-db, [Hyperledger Burrow](https://www.hyperledger.org/projects/hyperledger-burrow) was released in December 2014. Currently under incubation, Hyperledger Burrow is a permissionable smart contract machine that provides a modular blockchain client with a permissioned smart contract interpreter built- in part to the specification of the Ethereum Virtual Machine (EVM). It is the only available Apache-licensed EVM implementation.
+
+Following are the major components of Burrow:
+
+* The **Gateway** provides interfaces for systems integration and user interfaces
+* The **Smart contract application engine** facilitates integration of complex business logic
+* The **Consensus Engine** serves the dual purpose of:
+        a. Maintaining the networking stack between the nodes, and,
```

Comment:
These are not showing up as indented lines. I am guessing it is because markdown does not recognize a. and b. as list items. Maybe change to 1. and 2.